ionic/ngx/package.json

```
@@ -1,6 +1,6 @@
 {
-    "version": "7.7.0",
-    "name": "cordova-plugin-fcm-with-dependecy-updated",
+    "version": "7.8.1-penbase-beta-1",
```

```suggestion
    "version": "7.8.1-penbase",
```

C'est normal que tu conserves le suffixe `beta-1` dans ce fichier ?  Dans le `podspec`tu précises la version `7.8.1-penbase`.